src/Fixer/Operator/ConcatSpaceFixer.php

```diff
@@ -77,6 +79,29 @@ public function getDescription()
         return 'Concatenation should be spaced according configuration.';
     }
 
+    public function getFixerDescription()
+    {
+        return new FixerDescription(
+            array(
+                array(
+                    "<?php\n\$foo = 'bar' . 3 . 'baz'.'qux';",
+                    null,
+                ),
+                array(
+                    "<?php\n\$foo = 'bar' . 3 . 'baz'.'qux';",
+                    array('spacing' => 'none'),
+                ),
+                array(
+                    "<?php\n\$foo = 'bar' . 3 . 'baz'.'qux';",
+                    array('spacing' => 'one'),
+                ),
+            ),
+            "Configuration must have one element 'spacing' with value 'none' (default) or 'one'.",
```

Comment:
So, this is duplicating phpdoc for `configure` method.

Maybe after solving https://github.com/keradus/PHP-CS-Fixer/pull/18/files#r88816957 , when we will have same config-description format, we could autogenerate that part directly from `configure` docblock ?